packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -853,6 +854,8 @@ namespace KB = KokkosBatched::Experimental;
         }
 
         // 1. fire up all cuda events
+
+        // FENCE REVIEW: Not Tested (in #ifdef IFPACK2_BLOCKTRIDICONTAINER_USE_CUDA_STREAM)
         Kokkos::fence();
```

Comment:
@kyungjoo-kim This one too

Reply:
The stream version pack/unpack is not activated in the code. So, fence or no fence does not change the code behavior. I am actually working on this part and will handle this myself.